support/power-platform/power-automate/desktop-flows/invalid-credentials-errors-running-desktop-flows.md

```diff
@@ -50,6 +50,7 @@ To see an error message with specific details on what went wrong, ensure you hav
 |-1073741477|A user has requested a type of logon (for example, interactive or network) that has not been granted. An administrator has control over who can logon interactively and through the network.|The user account doesn't have logon rights on the machine, or the administrator changed the policies of the machine (check the [User Rights Assignment](/previous-versions/windows/it-pro/windows-10/security/threat-protection/security-policy-settings/user-rights-assignment) settings).|To solve this issue, see ["Logon type has not been granted" error when running a desktop flow or creating a connection](logon-type-has-not-been-granted.md).|
 |-1073445812|The user's UPN isn't in the expected format|The user signed in using `DOMAIN\user` but should use the `user@domain.com` format instead (or vice versa).|Try to sign in using both the `user@domain.com` and `DOMAIN\user` formats.|
 |-1073741062|Smart card logon is required and was not used| |Connections to machines that require smart card logons aren't supported. Use a machine without this requirement.|
+|-1073741715|Bad username or authentication information| Account must be checked on the device|Run the command `dsregcmd /status` in the context of the desktop flow connection account. Check the possible `Executing account names` section from the output and make sure it matches the username that has been used. Always prefer the upn  `user@domain.com` format on MSEntra joined or hybrid devices. Always use the domain format `DOMAIN\user` on domain-only joined devices|
```

Comment:
"Account must be checked on the device" --> isn't this supposed to be "Cause"? Shouldn't it be something like "The username couldn't be authenticated on the server side"? (actually not really sure what the cause is, but I don't think "account must be checked" is a cause? Sounds like an action to take... ?)

Also, for the last sentence, a suggestion: --> 
If your machine is MS Entra joined, use the "user@domain.com" format for your username. You should only use the "DOMAIN\user" format if your device is domain-only joined.

Reply:
Hi @alarnaudy @aartig13, per the comments from @johndund, I suggest updating LINE 53 to the following. Could you have a review?

|-1073741715|Bad username or authentication information| The "executing account name" or the format of your username is incorrect. |Run the [dsregcmd /status](/entra/identity/devices/troubleshoot-device-dsregcmd) command in the context of the desktop flow connection account. Check the `Executing Account Name` section from the output and make sure it matches the username that has been used. Always prefer the UPN  `user@domain.com` format on [Microsoft Entra joined](/entra/identity/devices/concept-directory-join) or hybrid devices. If your machine is Microsoft Entra joined, use the `user@domain.com` format for your username. You should only use the `DOMAIN\user` format if your device is domain-only joined.|

Reply:
Hi @alarnaudy @a-887178056, could you help double check the changes that I made for the draft? Thanks!